YearMonth.isValidDay() missing argument in typings

With `@js-joda/core` v5.6.4 and typescript, code like:

```
YearMonth.of(2025, 1).isValidDay(13)
```

...fails with `error TS2554: Expected 0 arguments, but got 1.`

It looks like the `js-joda.d.ts` file is missing the `dayOfMonth` parameter for this method.

--- a/packages/core/test/typescript_definitions/core-test.ts
+++ b/packages/core/test/typescript_definitions/core-test.ts
@@ -775,7 +775,7 @@ it('YearMonth', () => {
     ym.monthValue();
     ym.month();
     ym.isLeapYear();
-    ym.isValidDay();
+    ym.isValidDay(10);
     ym.lengthOfMonth();
     ym.lengthOfYear();
     ym.atDay(10);

--- a/packages/core/typings/js-joda.d.ts
+++ b/packages/core/typings/js-joda.d.ts
@@ -1579,7 +1579,7 @@ export class YearMonth extends Temporal implements TemporalAdjuster {
     isBefore(other: YearMonth): boolean;
     isLeapYear(): boolean;
     isSupported(fieldOrUnit: TemporalField | TemporalUnit): boolean;
-    isValidDay(): boolean;
+    isValidDay(dayOfMonth: number): boolean;
     lengthOfMonth(): number;
     lengthOfYear(): number;
     minus(amountToSubtract: number, unit: TemporalUnit): YearMonth;

--- a/packages/core/typings/js-joda.flow.js
+++ b/packages/core/typings/js-joda.flow.js
@@ -759,7 +759,7 @@ declare module "js-joda" {
         monthValue(): number;
         month(): Month;
         isLeapYear(): boolean;
-        isValidDay(): boolean;
+        isValidDay(dayOfMonth: number): boolean;
         lengthOfMonth(): number;
         lengthOfYear(): number;
         atDay(dayOfMonth: number): LocalDate;

